owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.kt

```diff
@@ -940,7 +930,7 @@ class FileDisplayActivity : FileActivity(),
                     FileListOption.ALL_FILES -> getString(R.string.default_display_name_for_root_folder)
                     FileListOption.SPACES_LIST -> getString(R.string.bottom_nav_spaces)
                 }
-            setupRootToolbar(title, isSearchEnabled = fileListOption != FileListOption.SPACES_LIST)
+            setupRootToolbar(title, isSearchEnabled = fileListOption != FileListOption.SPACES_LIST, false)
```

Comment:
Since we have 2 different boolean parameters now (and who knows if more in the future), let's add the parameter name to avoid confusions

Reply:
Added